airbyte-config/models/build.gradle

```diff
@@ -1,12 +1,16 @@
 import org.jsonschema2pojo.SourceType
 
 plugins {
+    id 'airbyte-docker'
```

Comment:
it seemed simpler to reuse the existing plugin, although it does add the airbyte-docker dependency. lmk if you think we should manually define this.

Reply:
Why is `airbyte-docker` required?

Reply:
it's a dependency of the java integration test plugin

Reply:
this has been removed.